### PR TITLE
Log non-string CIDs

### PR DESCRIPTION
**Pull request by jridgewell**

### CODE REVIEW

File: src/service/url-replacements-impl.js [hunk 1]
```diff
@@ -272,7 +272,12 @@ export class GlobalVariableSource extends VariableSource {
         // TODO: replace with "filter" when it's in place. #2198
         const cookieName = opt_cookieName || scope;
         if (cid && cookieName == '_ga') {
-          cid = extractClientIdFromGaCookie(cid);
+          if (typeof cid === 'string') {
+            cid = extractClientIdFromGaCookie(cid);
+          } else {
+            // Do not log the CID directly, that's PII.
```
Comment (erwinmombay) [inline src/service/url-replacements-impl.js]: can we add TODO to remove?

Comment (jridgewell) [reply src/service/url-replacements-impl.js]: Done.

**Comment by zhouyx** [comment]
I tried trace the non string cid this morning. It seems to me this can be the only promise that returns a JsonObject or string. Not sure if it helps.
https://github.com/ampproject/amphtml/blob/master/src/service/viewer-cid-api.js#L61

**Comment by cramforce** [comment]
+1. I looked at the code higher up and it doesn't seem to do that. Really
weird.

**Comment by lannka** [comment]
For #11120

**Pull request merged**